modules/ROOT/pages/hardening-your-mule-installation.adoc

```diff
@@ -0,0 +1,44 @@
+
+Normally, an IT organization already has control of keeping open ports to a minimum, restricting access to administrative applications, minimizing the number of applications, and other measures to ensure quality. We recommend using secure files to configure Mule, as well as using secure configuration files.
+
+== Hardening Checklist
+
+To begin hardening the Mule installation itself, see a simple list of steps:
+
+* Run Mule as a Non-privileged User
+* Install Mule as a Service
+* Make sure to configure Mule to write logs or temporary files within appropriate locations. Configure logs, passwords, and keystore files
+* In some situations, you need to configure usernames, passwords, and keystores on Mule. Usually, these settings are made available externally, so that dev/ops can change these settings.
+* Manage certificates in a keystore file
+* Use a separate property file to store usernames and passwords and secure it using file system permissions
+
+=== About Running Mule as a Non-Privileged User
+
+On Xnix, you can run Mule as any user with the following caveats:
+
+* You need write permissions to logs directory
+* Without root you cannot use ports below 1024
+
+On Windows, you need to be in admin user's group to run Mule.
+
+== Hardening in Layers
+
+By its nature, you can situate Mule in a variety of configurations. The suggested approach to hardening involves hardening in layers beginning with the operating system, then working up the stack. The Center for Internet Security (CIS) publishes configuration benchmarks that are widely used in whole or in part as system hardening guides. Mule TCat Server also offers added security options.
+
+Commercial configuration management and integrity management tools can help you automate management to the CIS benchmarks. Also, Mule documentation includes a good deal of information on configuring security. If your application deals in sensitive data, consider using SSL Transport (HTTPS) to protect it.
+
+On the network security side, security experts recommend using a good stateful inspection network firewall with a default-deny rule set and exceptions only for justified business needs. Also, any internet facing server belongs in a DMZ with strong default-deny egress rules on the firewall to prevent data exfiltration. Furthermore, you can use a network IDS/IPS to monitor and prevent known attacks. Putting the database on an internal network - not the DMZ - also helps to harden your installation.
+
+Be sure your software developers are familiar with secure web application coding techniques. At the very least, they should be familiar with best practices to avoid common web app pitfalls.
+
+== See Also
+* https://github.com/vagoff/xinx[Xnix GitHub]
```

Comment:
It's a link to Programming Language XinX, not Xnix. 
Also I think "Unix-like" is better wording than "Xnix" for the official documentation.

Reply:
Thank you. We'll update accordingly.

Reply:
@kevintroller I'm sorry for the confusion. I tried to suggest to:

* Replace "On XinX" with "On Unix-like OS"
* Remove the link to https://github.com/vagoff/xinx

> On XinX, you can run Mule as any user with the following caveats:

> On Windows, you need to be in admin user’s group to run Mule.

The above context, I interpreted Xnix (and XinX) as Unix based OS including Linux. I think Xnix is too casual. "Unix-like OS" is better wording for formal documentation.

XinX is a Programming Language (Clojure based?), not OS. So I don't think it's the correct link.

Here are references:

https://www.yourdictionary.com/xnix
https://en.wikipedia.org/wiki/Unix-like